pages/integrations/index.tsx

```diff
@@ -220,6 +220,13 @@ export async function getServerSideProps(context) {
         title: "Office 365 / Outlook.com Calendar",
         imageSrc: "integrations/office-365.png",
         description: "For personal and business accounts",
+    }, {
+        installed: !!(process.env.ZOOM_CLIENT_ID && process.env.ZOOM_CLIENT_SECRET),
+        type: "zoom_video",
+        credential: credentials.find( (integration) => integration.type === "zoom_video" ) || null,
+        title: "Zoom",
+        imageSrc: "integrations/zoom.png",
+        description: "For personal and business accounts",
```

Comment:
@PeerRich Do you think this is the best description to use for Zoom?

Reply:
I would just do "Video Conferencing", at least that's what Zoom says on their website